services/credential-server/src/apis/contact.api.ts

```diff
@@ -1,16 +1,35 @@
-import { Request, Response } from "express";
+import { NextFunction, Request, Response } from "express";
 import { Agent } from "../agent";
 import { ResponseData } from "../types/response.type";
 import { httpResponse } from "../utils/response.util";
 
-async function contactList(_: Request, res: Response) {
-  const data = await Agent.agent.contacts();
-  const response: ResponseData<any> = {
-    statusCode: 200,
-    success: true,
-    data,
-  };
-  httpResponse(res, response);
+async function contactList(_: Request, res: Response, next: NextFunction) {
+  try {
+    const data = await Agent.agent.contacts();
+    const response: ResponseData<any> = {
+      statusCode: 200,
+      success: true,
+      data,
+    };
+    httpResponse(res, response);      
+  } catch (error) {
+    return next(error);
```

Comment:
Is there no express way to auto-catch these? Seems weird that we have to manually put a try catch in every function

Reply:
https://expressjs.com/en/guide/error-handling.html - seems like express 5 supports auto passing to next - can you verify this? it would be much more maintainable and cleaner

Reply:
Express 5 is still in the beta stage. I think we should keep this at the moment, will update this later when the new version is stable. What do you think?
https://expressjs.com/en/guide/migrating-5.html

Reply:
I think we can still try - this is such a simple server and it's only for testing. If it seems stable we can use it - docker will auto restart anyway